ask-sdk-servlet-support/src/com/amazon/ask/servlet/verifiers/SkillRequestTimestampVerifier.java

```diff
@@ -55,6 +58,18 @@ public SkillRequestTimestampVerifier(long toleranceInMilliseconds) {
         this.toleranceInMilliseconds = toleranceInMilliseconds;
     }
 
+    /**
+     * Constructs a new timestamp verifier with the provided tolerance and timeUnit.
+     *
+     * @params tolerance, timeUnit
```

Comment:
Each param should be defined individually. When combined this way, the Javadoc generator can't figure them out and results in a build failure (see TravisCI).